### ISSUE TITLE
Celestica switch live ports.

### ISSUE THREAD

**Issue by jgonso07** [opensolutions/OSS_SNMP#77]
Hi Barry,
I have an issue with  a Celestica switch with Sonic OS. I can created via SNMP into the IXP, but I was unable to view the live ports and other features. I manually created the vendor_celestica.php, but still can't  view the ports. 
Could you help me please.
![imagen](https://user-images.githubusercontent.com/111540137/185511720-50fc447a-dd56-4587-b2b8-89e9b9612017.png)

**Comment by nickhilliard** [comment]
can you put vendor_celestica.php up on https://gist.github.com/?

**Comment by jgonso07** [comment]
Hi Nick, I already put the file. Let me know if you need anything else.
Best regards

**Comment by nickhilliard** [comment]
reference: https://gist.github.com/Jgon46/6a260e7178332837b68e25d42dfcfc20

Does IXP Manager recognise this as a `Seastone-DX010` unit?  If it does, what is the output of:

```
snmpwalk -v 2c -c <community> <switchname> ifIndex
snmpwalk -v 2c -c <community> <switchname> ifName
snmpwalk -v 2c -c <community> <switchname> ifDescr
snmpwalk -v 2c -c <community> <switchname> ifAlias
```

**Comment by jgonso07** [comment]
Nope, always show Unknown from the IXP and the snmpwalk as well, but if I
use the OID number related to IfIndex it works. [image: image.png]

**Comment by jgonso07** [comment]
Hi Nick, Do you have any suggestions or tests about this case?
Best regards.

**Comment by nickhilliard** [comment]
this switch is running SONiC, so it should be picked up by vendor_sonic.php.  Can you delete vendor_celestica.php?

What happens when you execute these commands from the OSS_SNMP directory:

```
% ./examples/platform.php <switchip> <community>
% ./examples/interfaces.php <switchip> <community>
```

**Comment by nickhilliard** [comment]
this means that OSS_SNMP is working fine, so I'm closing this issue on GH.

If there's nothing appearing in IXP Manager, then this might be an IXP Manager problem.  Can you email ixpmanager@inex.ie with the output of: 

```
./artisan switch:snmp-poll --log -vvv <switchname>
```

**Comment by jgonso07** [comment]
Thanks Nick, I will do it.
